persist/fs/mmap.go

```diff
@@ -27,6 +27,9 @@ import (
 	xerrors "github.com/m3db/m3x/errors"
 )
 
+// Package-level global for easy mocking
+var mmapFdFunc = mmapFd
```

Comment:
nit: For consistency make this `mmapFdFn`, we use `fn` as a suffix everywhere vs `func`